internal/docker/labels.go

```diff
@@ -33,7 +33,7 @@ func asIDToRegistrationFromLabels(labels map[string]string) map[string]string {
 	asMap := make(map[string]string)
 	for k, v := range labels {
 		if strings.HasPrefix(k, "application_service_") {
-			asMap[strings.TrimPrefix(k, "application_service_")] = v
+			asMap[strings.TrimPrefix(k, "application_service_")] = strings.Replace(v, "\\n", "\n", -1)
```

Comment:
Why did you change the format of `\n` and then do a replace over it?

Reply:
Because the `change` is a Dockerfile directive and it seems like it can't be on several lines.
This allows generating this kind of change `LABEL "mylabel"="multiline\nvalue"`.

Reply:
I added comments in the code regarding that for clarity.